File: CURSO PHP POO/CURSO POO PHP/Aula11/Tecnico.php
<?php
require_once 'Aluno.php';
class Tecnico extends Aluno {
    private $registroProfissional;
    
    public function praticar() {
        echo "<p> $this->nome ira praticar ".$this->getCurso();
        echo "</p>";
    }
    function getRegistroProfissional() {
        return $this->registroProfissional;
    }

    function setRegistroProfissional($registroProfissional) {
        $this->registroProfissional = $registroProfissional;
    }


}

File: EXERCÍCIOS JAVA E JAVA POO/Aula11_POO/src/aula11_poo/Tecnico.java
package aula11_poo;
public class Tecnico extends Aluno {
    private int registroProfissional;
    
    public void praticar(){
        System.out.println(this.nome+" irá praticar "+this.getCurso());
    }

    public int getRegistroProfissional() {
        return registroProfissional;
    }

    public void setRegistroProfissional(int registroProfissional) {
        this.registroProfissional = registroProfissional;
    }
    
}
